### PR TITLE
Added the final submission method for all the FA Survey sections

### PR DESCRIPTION
**Pull request by sonika-nih**
Added the final submission method for all the FA Survey sections

### CODE REVIEW

**Review by alenaminsk — APPROVED** [review]

**Review by Mariachaudhry — APPROVED** [review]

File: src/test/java/ServiceNow/CHARMS/Playwright/Features/FanconiStudy/FanconiSurveyEndToEndWorkFlow.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    When a Participant is on the Fanconi Study login page and logs with user credentials via Okta
+    Then fills the Background Information section for scenario 1
+    And fills the Demographic section for scenario 1
+    And fills the Medical History section for scenario 1
+    And fills the Medical History Continued section for scenario 1
+    And fills the Reproductive History section for scenario 1
+    And fills the Surgical and other Medical History section for scenario 1
+    And fills the Sexual History section for scenario 1
+    And submit the Fanconi Survey
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Playwright/Features/FanconiStudy/FanconiSurveyEndToEndWorkFlow.feature:19]: I would remove the empty line 19 but it's not a show stopper.

File: src/test/java/ServiceNow/CHARMS/Playwright/StepsImplementation/FanconiStudy/FanconiSurveyStepsImpl.java [hunk 2]
```diff
@@ -294,8 +296,10 @@ public static void addConsentDataForCallCompleteInNativeView() {
         page.frameLocator("iframe[name=\"gsft_main\"]").getByLabel("Interpreter Language").fill("Spanish");
         page.frameLocator("iframe[name=\"gsft_main\"]").getByLabel("Interpreter Witness?").selectOption("1");
         page.frameLocator("iframe[name=\"gsft_main\"]").getByLabel("Interpreter Signed").selectOption("1");
-        page.frameLocator("iframe[name=\"gsft_main\"]").locator("[id=\"x_naci_family_coho_fcsms_consent\\.form_header\"]").getByRole(AriaRole.BUTTON, new Locator.GetByRoleOptions().setName("Save")).click();
+      //  page.frameLocator("iframe[name=\"gsft_main\"]").locator("[id=\"x_naci_family_coho_fcsms_consent\\.form_header\"]").getByRole(AriaRole.BUTTON, new Locator.GetByRoleOptions().setName("Save")).click();
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Playwright/StepsImplementation/FanconiStudy/FanconiSurveyStepsImpl.java:299]: The commented code can be removed if no longer in use.

File: src/test/java/ServiceNow/CHARMS/Playwright/StepsImplementation/FanconiStudy/FanconiSurveyStepsImpl.java [hunk 3]
```diff
@@ -330,15 +335,19 @@ public static void completeConsentInNativeView() {
...
     /** Login method to Fanconi Survey */
     public static void loginFanconiSurvey() {
         var page = PlaywrightUtils.page;
         page.navigate(TestProperties.FANCONI_TEST_URL);
         page.getByRole(AriaRole.LINK, new Page.GetByRoleOptions().setName(FanconiSurveyPage.LOGIN_BUTTON_LINK)).click();
-        page.getByLabel(FanconiSurveyPage.USERNAME).fill(FanconiSurveyPage.EMAIL);
-        page.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName(FanconiSurveyPage.NEXT)).click();
+        CommonUtils.sleep(2000);
+      //  page.getByLabel(FanconiSurveyPage.USERNAME).fill(FanconiSurveyPage.EMAIL);
+     //   page.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName(FanconiSurveyPage.NEXT)).click();
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Playwright/StepsImplementation/FanconiStudy/FanconiSurveyStepsImpl.java:350]: The commented code can be removed if no longer in use.

**Review by hamidnesar — APPROVED** [review]
Looks good!

**Review by sarwarahmed1 — APPROVED** [review]

**Pull request merged**